sphere-models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchFilterTypesIntegrationTest.java

```diff
@@ -346,11 +559,12 @@ public void filtersTimeSetAttributes() throws Exception {
     }
 
     @Test
-    public void filtersDateTimeSetAttributes() throws Exception {
+    public void dateTimeSetAttributesFilters() throws Exception {
         final ProductProjectionSearch termSearch = ProductProjectionSearch.ofStaged()
                 .withQueryFilters(model -> model.allVariants().attribute().ofDateTime(ATTR_NAME_DATE_TIME_SET)
                         .filtered().by(ZonedDateTime.parse("2002-10-12T23:06:10.204+00:00")));
```

Comment:
So we go back to the first interpretation I did from your comment ;)
I will extract this and other complex values. all of them will achieve the opposite goal, I think.